.pre-commit-config.yaml

```diff
@@ -27,6 +27,13 @@ repos:
         args: ["--extend-exclude=topostats/_version.py"]
         additional_dependencies: [flake8-print]
         types: [python]
+  # - repo: https://github.com/pycqa/pylint.git
```

Comment:
Currently `pylint` runs using the `local` repository which is on your system. This is because the `pylint` pre-commit action defined under this commented out section doesn't include all of the dependencies we require in TopoStats and so if used `pylint` complains it can't import modules.

I've not yet worked out a way around this since `pre-commit.ci` hosts its own versions of each of the Virtual Environments it uses to run the tests and so for now I've left it as a `repo: local` so that `pylint` is run locally when we make commits. And yes its why in the `.pre-commit-config.yaml` I have set it to `skip: [pylint]`.

Reply:
Ah yes, I've seen this error with dependencies in my current PR.